tmeister/static/src/toggles/create/create-toggle/create-toggle-modal.component.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import styles from './create-toggle-modal.css'
+import { useCreateToggle } from './create-toggle.hooks.js'
+
+export default function CreateToggleModal (props) {
+  const { hide, refetchToggles } = props
+  const [ newToggleName, setNewToggleName ] = useState('')
+  const [ createName, setCreateName ] = useState('')
+  const [ saving, saveCompleted ] = useCreateToggle(createName)
+  if (saveCompleted) {
+    refetchToggles()
+    hide()
+  }
+  return (
+    <div className="cps-modal">
+      <div className="cps-modal__screen"></div>
+      <div className="cps-modal__dialog cps-card">
+        <div className="cps-card__header cps-subheader-sm">
+          <span>
+            Create a new feature toggle
+          </span>
+          <a
+            className="cps-modal__dialog__close cps-icon cps-icon-close"
+            onClick={hide}
+          />
+        </div>
+        <div className="cps-card__body">
+          <div className="cps-form-group">
+            <label htmlFor="new_feature_name">
```

Comment:
I know, I know... Who cares? But **I** do. `htmlFor` value doesn't match with the input `id` value

Reply:
I'm glad you care, this bug has existed in TM for a long time